.github/ci.yaml

```diff
@@ -0,0 +1,18 @@
+name: Build code
+on:
+  push:
+    branches:
+    - main
+  pull_request:
+    branches:
+    - main
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Lint
+      uses: make fix lint
```

Comment:
I believe running `fix` will make local edits fixing the source file, so the `lint` invocation following it will not report any errors, so `fix` is redundant here.